microsoft-edge/devtools-guide-chromium/whats-new/2023/05/devtools-113.md

```diff
@@ -0,0 +1,164 @@
+
+Now in Microsoft Edge 113, the **Memory** tool automatically unminifies object names in the heap snapshot:
+
+![A heap snapshot in the Memory tool displays unminified object names](./devtools-113-images/unminified-object-name-memory-tool.png)
+
+See also:
+* [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md)
+* [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md)
+* [Take a snapshot](../../../memory-problems/heap-snapshots.md#take-a-snapshot) in _Record heap snapshots using the Memory tool_.
+
+
+<!-- ====================================================================== -->
+## The Issues tool has better support for keyboard navigation
+
+<!-- Subtitle: In previous versions of Microsoft Edge, certain Close buttons in the issues tool were not displayed when navigating via the keyboard. In Microsoft Edge 113, this issue has been fixed. -->
+
+In previous versions of Microsoft Edge, changing the **Severity** or **Browser** filters in the **Issues** tool displayed an info bar at the top of the **Issues** tool, containing a **Regenerate issues** button.  However, navigating this info bar with the keyboard caused the **Close** button to disappear.  In Microsoft Edge 113, this issue has been fixed:
+
+![Navigating the Issues tool with the keyboard no longer causes the Close button to disappear](./devtools-113-images/issues-tool-regenerate-infobar-close-button.png)
+
+See also:
+* [Filter issues](../../../issues/index.md#filter-issues) in _Find and fix problems using the Issues tool_
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+
+
+<!-- ====================================================================== -->
+## In Focus Mode, hovering over a tool no longer makes the title of each tool disappear in the Activity Bar
+
+<!-- Subtitle: With the Activity Bar in the horizontal orientation, you can now see the title of each tool and hover over the tool icon to see a tooltip. -->
+
+In previous versions of Microsoft Edge, in Focus Mode, hovering over a tool in the Activity Bar would cause all the titles of the tools in the Activity Bar to disappear.  In Microsoft Edge 113, this issue has been fixed.  When the Activity Bar is at the top, when you hover over tool icons to see the name of the tool in a tooltip, you can still see all of the titles of the tools in the toolbar:
+
+![Hovering over a tool icon in the Activity Bar produces a tooltip with the name of the tool](./devtools-113-images/focus-mode-tooltip.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## Context menus in DevTools better support themes from VS Code
```

Comment:
@zoherghadyali -- change "context menu" to "popup menu" throughout this section?

Reply:
How about just getting more specific? Let's just call it the:
**Customize and control DevTools** (![Customize and control DevTools icon](../media/customize-devtools-icon-light-theme.png)) menu

Then we don't have to use popup or context menu.

Reply:
revised, see latest https://github.com/MicrosoftDocs/edge-developer/pull/2548/files